src/Illuminate/Foundation/Vite.php

```diff
@@ -119,10 +141,9 @@ public function __invoke($entrypoints, $buildDirectory = 'build')
         static $manifests = [];
 
         $entrypoints = collect($entrypoints);
-        $buildDirectory = Str::start($buildDirectory, '/');
```

Comment:
I removed this as it wasn't needed. this was in place because our tests were mocking things incorrectly.